core/templates/dev/head/pages/exploration_player/PlayerPositionServiceSpec.js

```diff
@@ -19,7 +19,7 @@
 describe('Player position service', function() {
   beforeEach(module('oppia'));
 
-  var pts, pps, scof;
+  var pts = null, pps = null, scof = null;
```

Comment:
FYI we typically always declare on separate lines.

Reply:
Done.